src/main/java/org/gridsuite/ds/server/service/DynamicSimulationWorkerService.java

```diff
@@ -127,21 +134,24 @@ public void preRun(DynamicSimulationRunContext runContext) {
         super.preRun(runContext);
         DynamicSimulationParametersInfos parametersInfos = runContext.getParameters();
 
-        // get script and parameters file from dynamic mapping server
-        Script scriptObj = dynamicMappingClient.createFromMapping(runContext.getMapping());
+        // get parameters file from dynamic mapping server
+        Parameter parameterObj = dynamicMappingClient.getParameters(runContext.getMapping());
 
         // get all dynamic simulation parameters
-        String parametersFile = scriptObj.getParametersFile();
+        String parametersFile = parameterObj.parametersFile();
```

Comment:
In dynawaltz we have class ParametersSet so I dont want to use the same name to avoid confusion. The name 'Parameter' here does not means a resource entity but used as a wrapper object which contains a original mapping name and a parameters file content in xml string representation.

Reply:
NO ACTION